proposed/2021/DotnetNugetTrustedSigners.md

```diff
@@ -0,0 +1,239 @@
+
+USAGE:
+    dotnet nuget trust author [OPTIONS] <name> <package>...
+
+EXAMPLE:
+    dotnet nuget trust author Contoso ./contoso.library.nupkg
+
+OPTIONS:
+        --allow-untrusted-root          Specifies if the certificate for the trusted signer should be allowed to chain to an untrusted root.
+    -h, --help                          Prints help information.
+    -v, --verbosity <LEVEL>             Set the MSBuild verbosity level. Allowed values are q[uiet], m[inimal], n[ormal], d[etailed], and diag[nostic].
+        --interactive                   Allows command to stop and wait for user input or action.
+        --configfile                    The NuGet configuration file. If specified, only the settings from this file will be used. If not specified, the hierarchy of configuration files from the current directory will be used.
+```
+
+##### NOTE
+
+For `trust author` and `trust repository`, `<package>` may be a glob/wildcard.
+This must be consistent across platforms, so the Windows version must expand
+these wildcards when it receives them as arguments.
+
+#### `> dotnet nuget trust repository`
+
+```
+Adds a trusted signer with the given name to the config, based on the repository signature(s) or countersignature(s) of one or more packages. If <name> already exists in the configuration, the signature(s) will be appended.
+
+
+USAGE:
+    dotnet nuget trust repository [OPTIONS] <name> <package>...
+
+EXAMPLE:
+    dotnet nuget trust repository Contoso ./contoso.library.nupkg
+
+OPTIONS:
+        --allow-untrusted-root          Specifies if the certificate for the trusted signer should be allowed to chain to an untrusted root.
+        --owners                        Semi-colon separated list of trusted owners to further restrict the trust of a repository.
+    -h, --help                          Prints help information.
+    -v, --verbosity <LEVEL>             Set the MSBuild verbosity level. Allowed values are q[uiet], m[inimal], n[ormal], d[etailed], and diag[nostic].
+        --interactive                   Allows command to stop and wait for user input or action.
+        --configfile                    The NuGet configuration file. If specified, only the settings from this file will be used. If not specified, the hierarchy of configuration files from the current directory will be used.
+```
+
+##### NOTE
+
+For `trust author` and `trust repository`, `<package>` may be a glob/wildcard.
+This must be consistent across platforms, so the Windows version must expand
+these wildcards when it receives them as arguments.
+
+#### `> dotnet nuget trust certificate`
+
+```
+Adds a trusted signer with the given name to the config, based on a certificate fingerprint.
+
+Note: If a trusted signer with the given name already exists, the certificate item will be added to that signer. Otherwise a trusted author will be created with a certificate item from given certificate information.
```

Comment:
As someone unfamiliar with this space, I do not know what name means. Is it part of the certificate?

Reply:
The name in all of these refers to the identifier the trusted signer is saved under. The name of the entity itself, if you will. It's an arbitrary name that we use to key off of.

Reply:
It's the key we save the signer under, in the config file, and what you use to refer to it later.